Fill.py

```diff
@@ -87,22 +98,6 @@ def distribute_items_restrictive(window, worlds, fill_locations=None):
         ice_trap = ice_traps.pop(0)
         ice_trap.looks_like_item = random_item
 
-    cloakable_locations = shop_locations + song_locations + fill_locations
-    all_models = shopitempool + dungeon_items + songitempool + itempool
-    worlds[0].settings.distribution.fill(window, worlds, [shop_locations, song_locations, fill_locations], [shopitempool, dungeon_items, songitempool, progitempool, prioitempool, restitempool])
-    itempool = progitempool + prioitempool + restitempool
-
-    ice_traps = [
-        location.item for location in cloakable_locations
-        if location.item is not None and location.item.name == 'Ice Trap' and location.item.looks_like_item is None]
-    fake_items = []
-    while len(ice_traps) > len(fake_items):
```

Comment:
Did you replace this loop elsewhere, or is it in your next PR?

Reply:
The diff is just confusing here. All of this still exists above, lines 64-99:
https://github.com/TestRunnerSRL/OoT-Randomizer/blob/8748832497393ce53a56156b2b147d96bfaa74c1/Fill.py#L64-L99

Reply:
To clarify, this loop was run twice with my old fix. This properly moves all of the ice trap handling after the distribution file fill instead of needlessly running essentially the same loop twice.

Reply:
Gotcha. Thanks!